File: p9.rb
#Author : Mackenzie Park-McInnes
#Title P9
#Purpose : Solve 'projecteuler.net' archived problem 9


for a in 1..900
	for b in a+1..900
		for c in b+1..900
			d = a ** 2 + b ** 2
			if d == c ** 2
				t = a+b+c
				if t == 1000
					puts (a*b*c)
					break
				end
			end
		end
	end
end

File: p9.py
#Author : Mackenzie Park-McInnes
#Title : P9
#Purpose : Solve 'projecteuler.net' archived problem 9

import sys

for a in range(1, 1000):
	for b in range(a+1, 1000):
		for c in range(b+1, 1000):
			sum = a**2 + b**2
			if sum == c**2:
				if a + b + c == 1000:
					print "%d\n%d\n%d" % (a, b, c)
					product = a * b * c
					print product
					sys.exit("Success")
